skopt/learning/gp_kernels.py

```diff
@@ -0,0 +1,186 @@
+from math import sqrt
+
+import numpy as np
+from sklearn.gaussian_process.kernels import Kernel as sk_Kernel
+from sklearn.gaussian_process.kernels import ConstantKernel as sk_ConstantKernel
+from sklearn.gaussian_process.kernels import DotProduct as sk_DotProduct
+from sklearn.gaussian_process.kernels import Exponentiation as sk_Exponentiation
+from sklearn.gaussian_process.kernels import ExpSineSquared as sk_ExpSineSquared
+from sklearn.gaussian_process.kernels import Matern as sk_Matern
+from sklearn.gaussian_process.kernels import Product as sk_Product
+from sklearn.gaussian_process.kernels import RationalQuadratic as sk_RationalQuadratic
+from sklearn.gaussian_process.kernels import RBF as sk_RBF
+from sklearn.gaussian_process.kernels import Sum as sk_Sum
+from sklearn.gaussian_process.kernels import WhiteKernel as sk_WhiteKernel
+
+
+class Kernel(sk_Kernel):
```

Comment:
No one inherits from this? It is good to have it in one place to define the API, but can we make use of it somehow?